{{cookiecutter.python_name}}/setup.py

```diff
@@ -2,28 +2,31 @@
 {{ cookiecutter.python_name }} setup
 """
 import json
-import os
+from pathlib import Path
 
 from jupyter_packaging import (
-    create_cmdclass, install_npm, ensure_targets,
-    combine_commands, skip_if_exists
+    create_cmdclass,
+    install_npm,
+    ensure_targets,
+    combine_commands,
+    skip_if_exists
 )
 import setuptools
 
-HERE = os.path.abspath(os.path.dirname(__file__))
+HERE = Path(__file__).parent.resolve()
 
 # The name of the project
-name="{{ cookiecutter.python_name }}"
+name = "{{ cookiecutter.python_name }}"
 
 # Get our version
-with open(os.path.join(HERE, 'package.json')) as f:
-    version = json.load(f)['version']
+with (HERE / "package.json").open() as f:
+    version = json.load(f)["version"]
```

Comment:
An option is to do this, and later use e.g. `url=pkg_data['homepage'],` instead of duplicating the data in here.

That way people only have to change it in one file if they change it after the fact.

```suggestion
pkg_data = json.loads((HERE / "package.json").read_bytes())
version = pkg_data["version"]
```

Reply:
Thanks for the suggestion. I would prefer not to apply here to focus on only switching from os to pathlib. Then when this is merged, #115 can be rebased - it will include that.

Reply:
ah whoops, I Forgot that existed